app/helpers/helpers/helpers_file.go

```diff
@@ -8,6 +8,39 @@ import (
 	"strings"
 )
 
+func GetCommand(line string) string {
+	return line[2:]
+}
+
+func IsTestFile(text string) bool {
+	return text == "t"
+}
+
+func CreateGolangTestFile(text string) string {
```

Comment:
are we passing here some text or a file name or a file path?
what if there is a typo and the file has two dots?
please change the way the file name / path is extracted to something safer

Reply:
addressed in d9c081de52e70f86c4ee0af1d15f3af69343fabf